sofl/onlinefix_game.py

```
@@ -77,258 +78,320 @@ def launch(self) -> None:
-        # Проверяем существование Proton
-        if not os.path.exists(proton_path):
+        # Use host Steam home when running inside Flatpak
+        steam_home = os.path.join(host_home, ".local/share/Steam")
+        proton_path = os.path.join(
+            steam_home, "compatibilitytools.d", proton_version, "proton"
+        )
+
+        # Проверяем существование Proton (check on host when in Flatpak)
+        try:
+            if in_flatpak:
+                check = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)